src/pages/home/report/ReportActionItemMessage.js

```diff
@@ -33,22 +32,24 @@ const defaultProps = {
 };
 
 function ReportActionItemMessage(props) {
-    const messages = _.compact(props.action.previousMessage || props.action.message);
-    const isAttachment = ReportUtils.isReportMessageAttachment(_.last(messages));
+    // TODO: Are all message "types" expected to have multiple fragments? Can we have a message with multiple attachment fragments?
+    const fragments = _.compact(props.action.previousMessage || props.action.message);
```

Comment:
> This makes me think that we probably should keep the margin-top on the View that wraps the fragments, instead of applying it to each fragment.

This is what I initially had in mind when I gave up the idea of moving the margin to `ReportActionItemFragment`. But currently, I'm becoming to feel that `ReportActionItemFragment` is ill-organized and that the app assumes that a `COMMENT` report action fragment is the only fragment of the report action's message.

In the code we modify, [there'a a line](https://github.com/Expensify/App/pull/25524/files#diff-b9ff030858e3f02f7df3df9722486ca2e56c4c875b1af163819d5f980c168bf6L37) like:

```js
const isAttachment = ReportUtils.isReportMessageAttachment(_.last(messages));
```

Now I feel that `_.last` means "the only", but _just in case_ let's say "the last".

Reply:
So what should I do, as a user, to observe such an action?